### PR TITLE
Added post about notebook submission tutorial

### PR DESCRIPTION
**Pull request by nicole-brewer**
## Checklist:

- [x] (Committee Chairs Only) I have posted the link for the PR in the usrse slack (#usrse-2023-conference-planning-committee) to ask for content reviewers
- [ ] I have previewed changes locally
- [ ] I have updated the README.md if necessary

cc @mrmundt @cabejackson @manning-ncsa @jmelot @jbteves @jscubida

### CODE REVIEW

File: _posts/2023-03-03-notebook-submission-tutorial.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: Notebook Submissions Tutorial
+date: 2023-03-03
+tags:
+---
+
+**Date:** April 7, 2023 
+**Time:** 3:00 PM EDT, 2:00 PM CDT, 1:00 MDT, 12:00 PM MST
```
Comment (mrmundt) [inline _posts/2023-03-03-notebook-submission-tutorial.md:9]: The times here don't make sense to me. Everything is in DT until 12PM MST?

File: _posts/2023-03-03-notebook-submission-tutorial.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+**Registration:** https://asu.zoom.us/meeting/register/tZIod-Ghpz4qGNb5O23aWxuKBadudH65Fquy
+
+[Submission of computational notebooks](https://us-rse.org/usrse23/submissions/notebooks/) are due Monday, May 1, 2023. To account for the broad variety of use cases for notebooks, we are accepting submissions in three categories, described in detail on the [the Notebooks Submissions](https://us-rse.org/usrse23/submissions/notebooks/) page. Submitters should use our [notebook templates](https://github.com/USRSE/notebook-templates) to submit. In order to improve the clarity of our expectations, we are hosting a tutorial session. We will walk through the technical details of using our templates to submit a notebook in R or Python. As this is our first year accepting notebook submissions, we anticipate this being a productive session with lots of Q&A. This session will be recorded for reference.
```
Comment (mrmundt) [inline _posts/2023-03-03-notebook-submission-tutorial.md:12]: Can you turn this into two separate paragraphs / maybe make a bulleted list of what attendees should expect to hear or learn?

**Review by mrmundt — CHANGES_REQUESTED** [review]

File: _posts/2023-03-03-notebook-submission-tutorial.md (hunk 1 above)
Comment (torwhite) [reply _posts/2023-03-03-notebook-submission-tutorial.md:9]: Arizona doesn't observe daylight savings, so this makes sense. Maybe we should add PDT though, so as not to leave out the Californians

**Comment by mrmundt** [comment]
@nicole-brewer - Can you please apply the requested changes? Thank you!

**Comment by nicole-brewer** [comment]
Requested changes made!

**Review by mrmundt — APPROVED** [review]

**Pull request merged**